Zen/Zen/NSObject_Extension.h

```diff
@@ -1,16 +0,0 @@
-//
-//  NSObject_Extension.h
-//  Zen
-//
-//  Created by Wojciech Czekalski on 15.06.2015.
-//  Copyright (c) 2015 Wojciech Czekalski. All rights reserved.
-//
-
-#import <Foundation/Foundation.h>
-
-@interface NSObject (Xcode_Plugin_Template_Extension)
```

Comment:
I didn't feel safe with this method being a category on NSObject. What if there are more plugins? What if this method is called on an `NSObject`?